site/content/en/docs/administration/advanced/analytics.md

```diff
@@ -154,6 +155,38 @@ Client events:
 
 <!--lint enable maximum-line-length-->
 
+### Working time calculation
+
+Here is a short overview of how CVAT deals with the user's working time:
+
+- The user interface collects events when a user interacts with the interface
+(resizing canvas, drawing objects, clicking buttons, etc)
```

Comment:
Please indent subsequent lines in each list item.

Reply:
I am not sure what you meant here, tried couple of ideas, but remark lint was not happy :/

Reply:
I meant:

```suggestion
  (resizing canvas, drawing objects, clicking buttons, etc)
```